yarn.lock

```diff
@@ -4871,9 +4871,9 @@ __metadata:
   linkType: hard
 
 "decimal.js@npm:^10.2.1":
-  version: 10.2.1
-  resolution: "decimal.js@npm:10.2.1"
-  checksum: 2931d044a01b423f6f449c2cadbf6ee0e738096cde3f2d9b4c392b7c0cbdc5a41961855f67c1ba7dbabf439ecd76a923b495d100866984a7bcaac4e225f12fe9
+  version: 10.4.3
```

Comment:
Lets revert the `yarn.lock` also, looks like these changes came from adding jsdom, but are now left in